sw/banshee/config/mempool.yaml

```diff
@@ -20,6 +20,13 @@ memory:
     start: 0x0
     end: 0x100000
     latency: 5
+  periphs:
```

Comment:
Did you use this config for your tests? I don't think it makes sense to modify the MemPool config, but rather create a new configuration for your tests.

Reply:
I reset the MemPool config file and added a new one for the test